Assets/BossRoom/Scripts/Client/Game/Action/AoeActionFX.cs

```diff
@@ -12,7 +12,7 @@ public AoeActionFX(ref ActionRequestData data, ClientCharacterVisualization pare
         public override bool Start()
         {
             base.Start();
-            m_Parent.OurAnimator.SetTrigger(Description.Anim);
+            m_Parent.TrySetTrigger(Description.Anim);
```

Comment:
why are we still trying to setTrigger client side? I see this in multiple files in this PR.